Formula/arduino-cli.rb

```diff
@@ -0,0 +1,34 @@
+class ArduinoCli < Formula
+  desc "Arduino command-line interface"
+  homepage "https://github.com/arduino/arduino-cli"
+  url "https://github.com/arduino/arduino-cli/archive/0.5.0-showports.tar.gz"
+  version "0.5.0-showports"
+  sha256 "96373b59e004f277474eb5a11e2a585633976c9939056b833e65e670e7393e77"
+  bottle :unneeded
+
+  depends_on "go"
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/arduino/arduino-cli").install buildpath.children
+
+    cd "src/github.com/arduino/arduino-cli" do
+      system "go", "build", "-o", bin/"arduino-cli"
+      prefix.install_metafiles
+    end
+  end
+
+  def caveats; <<~EOS
```

Comment:
Since this isn't specific to homebrew so shouldn't be a caveat. See https://docs.brew.sh/Formula-Cookbook#caveats

Reply:
done